app/views/budgets/investments/_author_actions.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if show_author_actions?(investment) %>
+  <div class="sidebar-divider"></div>
+  <div class="show-actions-menu">
+    <h2><%= t("budgets.investments.show.author") %></h2>
```

Comment:
Just to make sure it's not a typo: we used to have the `<h2>` line above the `show-actions-menu` line, and now it's below it. Is it OK this way?